SummerProject/static/src/router.js

```diff
@@ -1,12 +1,11 @@
 /* eslint max-len: ["error", { "ignoreStrings": true }]*/
 
 import React from 'react';
-import {Route, Switch, Redirect} from 'react-router-dom';
```

Comment:
why did you remove ```Redirect``` from imports?

Reply:
I forgot to add it back :(
Gonna fix it ASAP